miio/device.py

```diff
@@ -117,6 +117,8 @@ def __init__(
     ) -> None:
         self.ip = ip
         self.token = token
+        if hasattr(self, "timeout"):
```

Comment:
Any suggested modifications to the base class should be done in a PR of its own.

Reply:
done here: https://github.com/rytilahti/python-miio/pull/884